metadata.json

```diff
@@ -69,7 +69,7 @@
   "requirements": [
     {
       "name": "puppet",
-      "version_requirement": ">= 5.5.8 < 7.0.0"
+      "version_requirement": ">= 5.5.8 < 8.0.0"
```

Comment:
This does not drop support for Puppet 5 as the PR title suggest :wink:

Reply:
fixed it